thinkhazard/processing/import.py

```diff
@@ -110,7 +128,7 @@ def do_execute(self):
             )
         )
 
-    def import_shapefile(self, path):
+    def import_shapefile_ogr2ogr(self, path):
```

Comment:
Do you keep it for reference to get back to it once it's compatible with PG12 ?

Reply:
Yes, shp2pgsql setup take 200 Mo.
And Debian Sid has fixed GDAL 3.